src/malcolmWidgets/table/table.component.js

```diff
@@ -71,12 +71,111 @@ const styles = theme => ({
   },
 });
 
+const AlarmCell = props => (
+  <TableCell
+    className={
+      props.selectedRow === props.row
```

Comment:
this className calculation is done 3 times in this component, it might be useful to have a function for it (e.g. `tableCellClassName(props)`)